src/modules/billing/services/contacts-service.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const contactsConnector = require('../../../lib/connectors/crm-v2/contacts');
-const mappers = require('../mappers');
```

Comment:
Is there a reason for changing this pattern?

Reply:
The contacts mapper originally found in the billing module has been promoted up to the src/lib/mappers directory so it can be shared with the companies module.